dirs/dirs.go

```diff
@@ -357,6 +359,7 @@ func SetRootDir(rootdir string) {
 	SysfsDir = filepath.Join(rootdir, "/sys")
 
 	FeaturesDir = filepath.Join(rootdir, snappyDir, "features")
+	RunMnt = filepath.Join(rootdir, "/run/mnt")
```

Comment:
can we have a new line before this one, also I suppose there are quite a few places to use this now?